source/geod24/concurrency.d

```diff
@@ -1133,7 +1027,6 @@ struct ThreadInfo
             _send(MsgType.linkDead, tid, ident);
         if (owner != Tid.init)
             _send(MsgType.linkDead, owner, ident);
-        unregisterMe(); // clean up registry entries
```

Comment:
Can you comment in the commit message why this part was removed?

Reply:
Any update?

Reply:
I did add it to the commit